### PR TITLE
docs(guide): add information about static parameters.

### LINKED ISSUE #77: can't pass table name from variable without `{}`
### Environment

- Operating System: `Darwin`
- Node Version:     `v20.10.0`
- Nuxt Version:     `3.11.1`
- CLI Version:      `3.11.1`
- Nitro Version:    `2.9.5`
- Package Manager:  `bun@1.1.3`
- Builder:          `-`
- User Config:      `css`, `devtools`, `modules`, `fonts`, `shadcn`, `eslint`, `experimental`, `nitro`
- Runtime Modules:  `@nuxt/fonts@0.5.1`, `@nuxtjs/tailwindcss@6.11.4`, `@pinia/nuxt@0.5.1`, `shadcn-nuxt@0.10.2`, `@nuxt/eslint@0.3.0-beta.6`
- Build Modules:    `-`

### Reproduction

I will add one here soon.

### Describe the bug

```ts
// DOESNT WORK
// routes/tables/[name].get.ts
export default defineEventHandler(async (event) => {
  try {
    const db = useDatabase()
    const name = getRouterParam(event, 'name')
    const query = getQuery(event)
    const limit = (query?.limit ?? 10) as number

    const table = await db.sql`SELECT * FROM ${name} LIMIT ${limit}`

    return table.rows ?? []
  }
  catch (error) {
    createError({
      status: 500,
      name: 'ServerError',
      cause: 'Internal Server Error',
    })
  }
})
```

```ts
// DOES WORK
// routes/tables/[name].get.ts
export default defineEventHandler(async (event) => {
  try {
    const db = useDatabase()
    const name = getRouterParam(event, 'name')
    const query = getQuery(event)
    const limit = (query?.limit ?? 10) as number

    const table = await db.sql`SELECT * FROM {${name}} LIMIT {${limit}}`

    return table.rows ?? []
  }
  catch (error) {
    createError({
      status: 500,
      name: 'ServerError',
      cause: 'Internal Server Error',
    })
  }
})
```

### Additional context

It seems to work fine if I add the `{}` around the variable but maybe that was intended and isn't documented.

### Logs

```sh
near "?": syntax error

  at Database.prepare (node_modules/better-sqlite3/lib/methods/wrappers.js:5:21)
  at Object.prepare (node_modules/db0/connectors/better-sqlite3.mjs:24:29)
  at Object.sql (node_modules/db0/dist/index.mjs:34:38)
  at Object.handler (server/api/tables/[name].get.ts:9:1)
  at node_modules/h3/dist/index.mjs:1890:43
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async node_modules/h3/dist/index.mjs:1962:19
  at async Object.callAsync (node_modules/unctx/dist/index.mjs:72:16)
  at async Server.toNodeHandle (node_modules/h3/dist/index.mjs:2249:7)
```

### PR DESCRIPTION
**Pull request by amandesai01**
### 🔗 Linked issue

Resolves #77 

### ❓ Type of change
- [x] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
This pull request adds information on using Static Parameters.

### 📝 Checklist
- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by amandesai01** [comment]
I have added text as per my understanding and would suggest maintainers to update as per requirement.

**Comment by amandesai01** [comment]
@pi0 are you going to merge this? I see fair amount of people using string literals for tablename and getting into issues.

**Comment by hareland** [comment]
This should be documented -> Is it possible to approve this PR @pi0 ?

File: docs/1.guide/1.index.md [hunk 1]
```diff
@@ -40,8 +40,16 @@ await db.sql`INSERT INTO users VALUES (${userId}, 'John', 'Doe', '')`;
...
 const { rows } = await db.sql`SELECT * FROM users WHERE id = ${userId}`;
 console.log(rows);
+
+// Using static parameters
+const tableName = "users";
+const { rows } = await db.sql`SELECT * FROM {${tableName}} WHERE id = ${userId}`;
+console.log(rows);
 ```
 
+> [!NOTE]
```
Comment (pi0) [inline docs/1.guide/1.index.md:50]:
```suggestion
> [!IMPORTANT]
```

**Review by pi0 — APPROVED** [review]
Thanks ❤️

**Pull request merged**